storage/persistent_storage/persistent-storage-aws.adoc

```diff
@@ -6,24 +6,50 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
+ifdef::openshift-rosa,openshift-dedicated[]
+{product-title} (ROSA) clusters are prebuilt with four storage classes that use AWS Elastic Block Store (EBS) volumes. These storage classes are ready to use and some familiarity with Kubernetes and AWS is assumed.
```

Comment:
**(ROSA)** is not applicable to  OSD cluster

Reply:
Fixed this.